mastermind.rb

```diff
@@ -1,25 +1,50 @@
 class Mastermind
+  attr_reader :red_count, :white_count
+
   def print_instructions
-    print "Welcome to Mastermind! The object of the game is to guess a 4-color code "\
-        "within 10 guesses. The available colors are red, orange, yellow, green, "\
-        "blue, and purple. Colors may be used in the code more than once or not at all."\
+    puts "\tWelcome to Mastermind! The object of the game is to guess a 4-color code
+        within 10 guesses. The available colors are red, orange, yellow, green,
+        blue, and purple. Colors may be used in the code more than once or not at all.
+
+        To make a guess: type the first letter of each color you're guessing in caps.
+        For example, if you want to guess 'red, orange, yellow, red,' type 'ROYR'.
 
-        "\nTo make a guess: type the first letter of each color you're guessing in caps. "\
-        "For example, if you want to guess 'red, orange, yellow, red,' type 'ROYR'."\
+        After each guess, you will receive feedback in the form of colored pins,
+        letting you know:
+          \t- how many colors are correct but in the wrong position (red pins)
+          \t- how many colors are correct and in the right position (white pins)
 
-        "\nAfter each guess, you will receive feedback in the form of colored pins, letting you know:"\
-          "\n\t- how many colors are correct but in the wrong position (white pins)"\
-          "\n\t- how many colors are correct and in the right position (red pins)"\
+        To quit the game, type '/q'. To start a new game, type '/r'."
+  end
 
-        "\nTo quit the game, type '/q'. To start a new game, type '/r'."
+  def color_count(code)
+    @color_count = Hash.new(0)
```

Comment:
Ah yes, thanks, Pat. I was originally thinking of having it as a get-able attributes in case another class one day wants to get the count...maybe that's not needed though and they can just call the `color_count` method to get it.

Reply:
Yeah, I usually like wait until something is needed before I make a code change; anticipating requirements is a risky strategy in software development.